packages/components/psammead-sitewide-links/src/List/index.jsx

```diff
@@ -73,17 +73,13 @@ const StyledList = styled.ul`
     padding: ${GEL_SPACING} 0;
     margin-bottom: ${GEL_SPACING};
     grid-column: 1/-1;
-    @supports not (${grid}) {
-      width: 100%;
-    }
+    width: 100%;
```

Comment:
Wondering if we would also need this:
```
@supports (${grid}) {
    width: initial;
}
```

Reply:
I could add this just to be safe but it doesn't look like the component is affected by this change if you look at the screenshots above

Reply:
Yep, just tested and it works fine without this, as when supporting grid the first list item will take the whole column 👍🏻